documentation/slate/source/includes/apis.md

```diff
@@ -1006,6 +1008,18 @@ Examples:
 * A user doesn't like line A metro in hers city. She adds the parameter `forbidden_uris[]=line:A` when calling the API.
 * A user would only like to use Buses and Tramways. She adds the parameter `allowed_id[]=physical_mode:Bus&allowed_id[]=physical_mode:Tramway`.
 
+### Precisions on `free_radius_from/free_radius_to`
+
+These parameters allow to find the nearest stop point (within free_radius distance) to the given coordinates.
```

Comment:
I'd put it like this (remove "allow to", add "Then it" + "skipping" + "those nearest stop points"):
```
These parameters find the nearest stop point (within free_radius distance) to the given coordinates.
Then it allows skipping walking sections between the point of departure/arrival and those nearest stop points.
```